magento/pwa-studio
# fix: Add youtube and vimeo urls to CSP for PageBuilder

## Description

Youtube and Vimeo URLS would be blocked by our previous CSP. Add them.

## Related Issue





Closes [PWA-842](https://jira.corp.magento.com/browse/PWA-842).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Set a back end with PageBuilder/Youtube videos as your `MAGENTO_BACKEND_URL`.
2. `yarn build && yarn stage:venia`
3. Open the local home page and verify there is no console error.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
